automated-testing/synthetic-monitoring-tests/readme.md

```diff
@@ -0,0 +1,82 @@
+# Synthetic Monitoring Tests
```

Comment:
Is this class of tests also known as "Watchdog tests"?  If yes, is it useful to point out the synonym?

Reply:
I found very little online references to this synonym. most were referring to HW/IoT based service monitoring.
Wasn't sure if to add it or not. wdyt?

Reply:
Another term I have heard is "synthetic transactions"

Reply:
added